ArchiSteamFarm/Steam/Data/AccessTokenResponse.cs

```diff
@@ -0,0 +1,17 @@
+
+using System.Diagnostics.CodeAnalysis;
+using Newtonsoft.Json;
+
+namespace ArchiSteamFarm.Steam.Data {
+	[SuppressMessage("ReSharper", "ClassCannotBeInstantiated")]
+	internal sealed class AccessTokenResponse : EResultResponse {
+		[JsonProperty(PropertyName = "data", Required = Required.Always)]
+		internal AccessTokenData AccessTokenData { get; private set; } = new();
+	}
+
+	[SuppressMessage("ReSharper", "ClassCannotBeInstantiated")]
+	internal sealed class AccessTokenData {
```

Comment:
Consider moving this class into a separate file.

Reply:
Internal classes of the main do not need standalone parts, otherwise we'll end up in a load of trash, see `UserPrivacy`.

Reply:
`UserPrivacy` contains nested types, whereas `AccessTokenResponse` and `AccessTokenData` are two separate types.

Reply:
Ah right, you're correct then, sorry 🙂

Reply:
It's possible to make `AccessTokenData` nested, because it's not used in any other places - that will require renaming of `AccessTokenData` property though.